CHANGES.md

```diff
@@ -136,7 +136,7 @@
 
 ## Known Issues
 
-* ([BEAM-X](https://issues.apache.org/jira/browse/BEAM-X)).
+* Users of beam-sdks-java-io-hcatalog (and beam-sdks-java-extensions-sql-hcatalog) must take care to override the transitive log4j dependency when they add a hive dependency ([BEAM-13499](https://issues.apache.org/jira/browse/BEAM-13499)).
```

Comment:
Do we still need this warning if we  CP https://github.com/apache/beam/pull/16289 ?

Reply:
CP is here: https://github.com/apache/beam/pull/16302

Reply:
Yes, unfortunately that change only affects Beam's build for running tests. To use these packages users must also specify their own Hive dependency (since in Beam it's `provided`), so we don't have control over the transitive log4j dependency.